apps/my-vehicles/package.json

```diff
@@ -21,11 +21,14 @@
   },
   "dependencies": {
     "@zodios/core": "^10.9.6",
+    "i18next": "^23.15.1",
```

Comment:
is this used somewhere?

Reply:
Yes, indirectly, it is necessary for the correct behaviour of next-i18next library.
In fact, if you remove it, you will get the following runtime error:

> ./node_modules/next-i18next/dist/esm/createClient/browser.js:1:1
> Module not found: Can't resolve 'i18next'

Reply:
https://github.com/i18next/next-i18next/tree/master?tab=readme-ov-file#1-installation